src/lib/travatar-runner.cc

```diff
@@ -76,11 +76,21 @@ void TravatarRunnerTask::Run() {
     if(nbest_collector_ != NULL) {
         ostringstream nbest_out;
         BOOST_FOREACH(const boost::shared_ptr<HyperPath> & path, nbest_list) {
-            nbest_out
-                << sent_
-                << " ||| " << Dict::PrintWords(path->GetTrgData())
-                << " ||| " << path->GetScore()
-                << " ||| " << Dict::PrintSparseVector(path->CalcFeatures()) << endl;
+            if(runner_->GetNbestWithTree() == false){
```

Comment:
The "if" statement here doesn't need to be around all 5 lines. Could you put it just around the line "<< " ||| " << path->GetTreeStr()"